ethereum/client/src/lib.rs

```diff
@@ -358,72 +430,189 @@ impl Contracts {
 
 #[cfg(test)]
 mod tests {
```

Comment:
Please move tests to the separate file

Reply:
Tried to find example in this project without success.

Reply:
Done